test/members_test.exs

```diff
@@ -128,4 +128,53 @@ defmodule GlassFactoryApi.MembersTest do
       end
     end
   end
+
+  describe "list_active_members/1" do
+    test "returns a list of members", %{bypass: bypass, config: config} do
+      request_response = GlassFactoryApi.Fixtures.Members.list()
```

Comment:
I think [this idea](https://github.com/Cendrao/glass_factory_api/pull/37#discussion_r352800016) applies here too.